README.md

```diff
@@ -43,7 +43,7 @@ Or install it yourself as:
 
 ### Ruby
 
-Chewy is compatible with MRI 2.6-3.0¹.
+Chewy is compatible with MRI 3.0-3.2¹.
 
 > ¹ Ruby 3 is only supported with Rails 6.1
```

Comment:
What does it mean?  Is it still relevant? Id Ruby 3.0/Rails 7 incompatible?

Reply:
ah, it means that ruby 3.0 starts support only with rails 6.1 :)